include/tbl/vsl_tags.h

```diff
@@ -107,27 +107,31 @@ SLTM(SessClose, 0, "Client connection closed",
 SLTM(BackendOpen, 0, "Backend connection opened",
 	"Logged when a new backend connection is opened.\n\n"
 	"The format is::\n\n"
-	"\t%d %s %s %s %s %s %s\n"
-	"\t|  |  |  |  |  |  |\n"
-	"\t|  |  |  |  |  |  +- \"connect\" or \"reuse\"\n"
-	"\t|  |  |  |  |  +---- Local port\n"
-	"\t|  |  |  |  +------- Local address\n"
-	"\t|  |  |  +---------- Remote port\n"
-	"\t|  |  +------------- Remote address\n"
-	"\t|  +---------------- Backend display name\n"
-	"\t+------------------- Connection file descriptor\n"
+	"\t%d %s %s %s %s %s %s %.6f %ld\n"
+	"\t|  |  |  |  |  |  |  |    |\n"
+	"\t|  |  |  |  |  |  |  |    +- Backend reuses\n"
+	"\t|  |  |  |  |  |  |  +------ Backend age in seconds\n"
+	"\t|  |  |  |  |  |  +--------- \"connect\" or \"reuse\"\n"
+	"\t|  |  |  |  |  +------------ Local port\n"
+	"\t|  |  |  |  +--------------- Local address\n"
+	"\t|  |  |  +------------------ Remote port\n"
+	"\t|  |  +--------------------- Remote address\n"
+	"\t|  +------------------------ Backend display name\n"
+	"\t+--------------------------- Connection file descriptor\n"
```

Comment:
I think I would rather refer to `Connection uses` and `Connection age` (everything is in seconds since 4.0 anyway).

Also, I would probably make these fields optional, they aren't really helpful in the "connect" case.